### PR TITLE
Add more descriptive error message for the CeleryExecutor

### PR DESCRIPTION
**Pull request by yyyyyyuanfei**
Dear Airflow Maintainers,

Please accept the following PR that
- add more instructions for using the CeleryExecutor.
- remove the `testdruid` DAG in the top level directory as it imports `HiveToDruidTransfer` which in turn imports `HiveOperator` that doesn't always exist.

Thanks!

### CODE REVIEW

**Comment by bolkedebruin** [comment]
I think you are a bit too eager in deleting files (eg. testdruid.py)

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -440,13 +440,23 @@ def serve_logs(filename):  # noqa
...
 
 
 def worker(args):
+    """
+    Spawn Celery workers for using the CeleryExecutor
+    """
+
+    # only applied when using the CeleryExecutor
+    executor_type = conf.get('core', 'executor')
+    if executor_type != 'CeleryExecutor':
```
Comment (jlowin) [inline airflow/bin/cli.py]: Maybe do a case-insensitive check:
`if conf.get('core', 'executor').lower() != 'celeryexecutor':`

File: airflow/executors/__init__.py [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
 from airflow.executors.sequential_executor import SequentialExecutor
-
-try:
-    from airflow.executors.celery_executor import CeleryExecutor
-except:
-    pass
-
 from airflow.exceptions import AirflowException
 
 _EXECUTOR = configuration.get('core', 'EXECUTOR')
```
Comment (jlowin) [inline airflow/executors/__init__.py]: I didn't realize until my previous comment that Airflow is enforcing capitalization for executor configuration. While you're working in this file, do you mind making that more lenient? Store `_EXECUTOR` as all lower case and do lower case comparisons thereafter.

**Comment by jlowin** [comment]
@bolkedebruin maybe the `testdruid.py` file shouldn't be deleted but it I have always found it a little confusing that it sits at the top-level in its own "dags" folder. It isn't used anywhere in Airflow. Maybe it should move to example_dags? @mvfast does the file serve a special purpose for you? I think it's too specific to be useful to anyone else.

File: airflow/bin/cli.py (hunk 1 above)
Comment (yyyyyyuanfei) [inline airflow/bin/cli.py]: Sure, added a helper in configuration to get the executor type and cast it to lowercase.

**Comment by yyyyyyuanfei** [comment]
@bolkedebruin @jlowin I agree deleting it is a bit exaggerated, it's better to move it to the example dags, but it seems to serve the purpose of demonstrating the `HiveToDruidTransfer`. How about putting it into the inline example of that operator?

**Comment by bolkedebruin** [comment]
@stranbird That is fine to me however I would prefer that in a separate pr/commit. It has no relation with what you want to do.

**Comment by yyyyyyuanfei** [comment]
@bolkedebruin Ok, added back the file.

**Comment by artwr** [comment]
@yuanfeiz, as an effort to clean up old PRs I am taking a closer look. I am not sure how I feel about the added code to get the executor in lower case. I think it is fine to test against the CamelCase version since it is what we expect. if you really feel like it, you can test against `executor_type.lower()` whenever you need. Let me know if you want to rebase and continue with this PR. If we do not hear in a few days, we will close this as abandoned.

**Comment by yyyyyyuanfei** [comment]
Sure, it looks stale. Go ahead closing it.

**Pull request closed without merging**